frontend/packages/knative-plugin/src/components/add/EventSource.tsx

```diff
@@ -100,7 +102,14 @@ const EventSource: React.FC<Props> = ({
       onReset={history.goBack}
       validationSchema={eventSourceValidationSchema}
     >
-      {(props) => <EventSourceForm {...props} namespace={namespace} projects={projects} />}
+      {(props) => (
+        <EventSourceForm
+          {...props}
+          namespace={namespace}
+          projects={projects}
+          showSinkAlert={service?.loaded && !service?.data?.length}
```

Comment:
@invincibleJai Why not render the alert here in this component only? The alert doesn't seem to be related to the form in anyway.

Reply:
Done